Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[workspace]
+members = ["rust/operator-binary", "rust/csi-grpc"]
+
+[patch.crates-io]
+# Workaround for https://github.com/hyperium/tonic/issues/243
+h2 = { git = "https://github.com/stackabletech/h2.git", branch = "feature/grpc-uds" }
+
+[patch."https://github.com/stackabletech/operator-rs.git"]
+# stackable-operator = { git = "https://github.com/stackabletech//operator-rs.git", branch = "feature/listener-implied-labels" }
+# stackable-operator = { path = "../operator-rs" }
```

Comment:
I agree that it is convenient to have these comments ready for development. Feel free to remove them or keep them.